internal/component/database_observability/mysql/collector/schema_table.go

```diff
@@ -119,6 +150,21 @@ type columnSpec struct {
 	DefaultValue  string `json:"default_value,omitempty"`
 }
 
+type indexSpec struct {
+	Name     string   `json:"name"`
+	Type     string   `json:"type"`
+	Columns  []string `json:"columns"`
+	Unique   bool     `json:"unique"`
```

Comment:
my brain got twisted with the name of the variable `Unique` and the column name as `non_unique`. I'm sure you got it the right way, just want to double check with you that it's ok?

Reply:
What do you mean exactly? Later on this is assigned as:

```
Unique: nonUnique == 0
```

Reply:
I just wanted a double-check that it was assigned correctly as the sense of unique was switched between the column and the variable to which it was scanned, along with interpreting 0.

Got it now from the docs https://dev.mysql.com/doc/refman/8.4/en/information-schema-statistics-table.html
> 0 if the index cannot contain duplicates, 1 if it can.

Reply:
I've added a comment about this as well, later on when the value is assigned.